src/nvim/lua/executor.c

```diff
@@ -2063,12 +2063,13 @@ char *nlua_register_table_as_callable(const typval_T *const arg)
   return name;
 }
 
-void nlua_execute_on_key(int c, char *typed_buf)
+// @return true to discard the key
```

Comment:
```suggestion
/// @return true to discard the key
```

Reply:
Right. Since this is private, I probably thought `///` might have a bad interaction with the public documentation generation...

Reply:
DONE